rules/what-to-do-when-a-client-says-no/rule.md

```diff
@@ -0,0 +1,89 @@
+- **Cost concerns** – The client sees your solution as too expensive.
+- **Timing issues** – A business isn't ready to implement the suggested solution.
+- **Misalignment with needs** – A client doesn’t see how the solution fits into their business.
+- **Unclear value proposition** – The client doesn’t understand the return on investment.
+- **Preexisting solution** - The client has an existing solution that currently meets their needs.
+
+Instead of immediately trying to resolve their issues with the solution, **ask questions** to understand their real hesitation:
+::: greybox
+"We can use a cheaper LLM to ensure the solution is within you're budget"
+:::
+::: Bad  
+Figure: Bad example – Trying to resolve the issue by assuming the customers concerns.
+:::
+
+::: greybox
+"Can you help me understand what concerns you have with the current approach?"
+:::
+::: good  
+Figure: Good example – Asking the right questions helps you refine your response.
+:::
+
+## Step 2: Return with a Better Argument
+
+Once you understand their objections, adjust your approach. Here’s how:
+
+### 1. **If cost is an issue: Show the ROI**
+   - Highlight long-term savings or increased revenue.
+   - Compare the cost of inaction.
+   - Offer a phased approach or a smaller-scale implementation.
+
+### 2. **If timing is an issue: Keep them engaged**
+   - Ask, “When would be a better time to revisit this?”
+
+### 3. **If they don’t see the fit: Provide better examples**
+   - Use case studies relevant to their industry.
+   - Show how similar clients have benefited.
+   - Incorporate the clients feedback and offer a more tailored solution.
+
+### 4. **If they don’t see the value: Clarify your messaging**
+   - Simplify your explanation—less jargon, more direct impact.
+   - Use visuals, data, or testimonials to reinforce your point.
+   - Focus on their pain points rather than just listing features.
+
+## Step 3: Present with Confidence
+
+When you go back, don’t just repeat your pitch **reframe it** based on what you’ve learned. Emphasise the value from their perspective, and make it clear you’re there to solve the problem.
+
+::: greybox
+“I understand your concern about cost, but let me show you how one of our clients in your industry reduced expenses by 20% within six months by using this approach.”
+:::
+::: good  
+Figure: Good example – Addressing concerns with data and real-world examples.
+:::
+
+## Step 4: Accept a Final No with Grace
+
+If the client still says no, accept it professionally. Leave the door open for future discussions:
+::: greybox
+"I appreciate your time and insights. If anything changes in the future, I’d love to revisit this conversation."
+:::
+::: good  
+Figure: Good example – Keeping the relationship positive for future opportunities.
+:::
+
+### Final Thought
+
+A no isn’t always the end—it’s often just the beginning of a better conversation. Listen, refine, and come back stronger like Steve Bucknor. 
```

Comment:
Hi @RobJT-wq 

- [ ] 1. I would quote "No" here
- [ ] 2. Maybe a wikipedia link to Steve Bucknor...